osstats/collector.go

```diff
@@ -155,6 +165,51 @@ func (s *StatsCollector) get() (*osStat, error) {
 		MemAvailableMB: formatMB(vm.Available), MemUsedMB: formatMB(vm.Used), SwapMemPct: swap.UsedPercent, CPUTotal: s.cpuTotal}, nil
 }
 
+func DumpProcessInfo() error {
+	// Retrieve list of processes
+	processes, err := process.Processes()
```

Comment:
given that this process list can be quite huge and since we are running every 10 minutes, might bring down le due to large size of logs or cause slowness. I wonder if we can do some filter or decrease the frequency.
Also init logs might get truncated due to this.

Reply:
Yeah that's why we are printing on demand only. Whenever stage variable is set or no logs are generated for 10 min we are dumping process logs